src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/BOM/MaterialForm/MaterialFormView.tsx

```diff
@@ -89,9 +90,9 @@ const MaterialFormView: React.FC<MaterialFormViewProps> = ({
               defaultValue={control._defaultValues.status}
               render={({ field }) => (
                 <TextField {...field} select variant="outlined" error={!!errors.status} helperText={errors.status?.message}>
-                  {['ORDERED', 'UNORDERED', 'SHIPPED', 'RECEIVED'].map((status) => (
+                  {['ORDERED', 'NOT_READY_TO_ORDER', 'SHIPPED', 'RECEIVED', 'READY_TO_ORDER'].map((status) => (
```

Comment:
you should use the shared type enum values here instead instead of raw string values